### DISCUSSION TITLE
Moose will not install on HPC, cannot connect to github

### DISCUSSION THREAD

**Discussion by jackeggemeyer** [idaholab/moose#27587] [Q&A General] [answered]
### Check these boxes if you have followed the posting rules.

- [X] Q&A General is the most appropriate section for my question
- [X] I have consulted the posting Guidelines on the Discussions front page
- [X] I have searched the Discussions forum and my question has not been asked before
- [X] I have searched the MOOSE website and the documentation does not answer my question
- [X] I have formatted my post following the posting guidelines (screenshots as a last resort, triple back quotes around pasted text)

### Question

I am new to this HPC/Sawtooth  and I am currenlty following this tutorial on installing moose on sawtooth: 

https://mooseframework.inl.gov/getting_started/installation/inl_hpc_install_moose.html

However; it seems like my terminal cannot access github to download moose. I have followed the instructions but cannot get past cloning moose. Here is what my terminal shows:
```
[eggejohn@sawtooth4 ~]$ qsub -I -P edu_res -l select=1:ncpus=20:mem=16gb -l walltime=1:00:00
qsub: waiting for job 1117673.sawtoothpbs to start
qsub: job 1117673.sawtoothpbs ready

(base) [eggejohn@r3i1n14 ~]$ module load use.moose moose-dev
(base) [eggejohn@r3i1n14 ~]$ mkdir -p ~/Sawtooth/projects
(base) [eggejohn@r3i1n14 ~]$ cd ~/Sawtooth/projects
(base) [eggejohn@r3i1n14 projects]$ clone https://github.com/idaholab/moose.git
-bash: clone: command not found
(base) [eggejohn@r3i1n14 projects]$ git clone https://github.com/idaholab/moose.git
Cloning into 'moose'...
fatal: unable to access 'https://github.com/idaholab/moose.git/': Failed to connect to github.com port 443: Connection timed out
(base) [eggejohn@r3i1n14 projects]$ 
```
What are my next steps? I assume im doing something small worng with internet connection or something but my browser in the same interactive section can access the web. Should I do this in a login node or something?

**Comment by GiudGiud** [comment, marked as answer]
Hello

compute nodes on sawtooth do not have access to the internet
You ll need to install on the head node

Guillaume

**Reply by jackeggemeyer** [reply]
Ah thank you that fixed it. That's the opposite to what I am used too
